WordPress/wporg-mu-plugins
# Tools: Create a workflow to push built files to a `build` branch

Fixes #136 — Uses the same build script as pattern-directory to create a new branch, `build`, with the built files force-added (to work around the `.gitignore`).

To do: Fix the cache directory - copied over from the yarn steps but something must be different with NPM.

**To test**

- The new branch `build` has been created by the action: https://github.com/WordPress/wporg-mu-plugins/tree/build
- You can see the build folder inside the blocks directory now exists and contains the processed CSS files: https://github.com/WordPress/wporg-mu-plugins/tree/build/mu-plugins/blocks/global-header-footer/build

~Update: anyone else can feel free to grab this (assign yourself), I'll circle back to it when I can if no one else does.~
